app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,165 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+/* universal */
+body {
+padding-top: 60px;
```

```suggestion
  padding-top: 60px;
```
tương tự với các dòng khác nha